src/oneapi/oneapi_backend.cpp

```diff
@@ -43,25 +51,25 @@ void PySyclExecutionContext::apply()
 }
 
 #if INTEL_DAAL_VERSION >= 20210200
-inline const sycl::queue& get_current_queue()
+inline const sycl::queue & get_current_queue()
 {
-    auto& ctx = daal::services::Environment::getInstance()->getDefaultExecutionContext();
-    auto* sycl_ctx = dynamic_cast<daal::services::internal::sycl::SyclExecutionContextImpl*>(&ctx);
-    if (!sycl_ctx)
+    auto & ctx     = daal::services::Environment::getInstance()->getDefaultExecutionContext();
+    auto * syclCtx = dynamic_cast<daal::services::internal::sycl::SyclExecutionContextImpl *>(&ctx);
```

Comment:
Again, this file was modified by old clang-format style. Is it done with some intent?

Reply:
these are old interfaces. why apply the new clang format to it?

Reply:
Ok